client/src/ingredients/attributes.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+function RenderValues({ values }) {
+  return (
+    <ul>
+      {values.map(value => {
+        return (
+          <li key={value.value}>
+            <code>{value.value}</code>:{" "}
+            <div dangerouslySetInnerHTML={{ __html: value.description }} />
```

Comment:
Like we said, this isn't great. The DOM becomes:
```html
  <li>
    <code>Foo</code>:
    <div><p>Bla bla</p></div>
  </li>
```
to start, I think the `<code>` tag should be in its own block, and without the `:`. E.g. 

```html
  <li>
    <p><code>Foo</code></p>
    <div><p>Bla bla</p></div>
  </li>
```